docs/src/main/asciidoc/_developer-guide/JTA_Reference.adoc

```diff
@@ -169,7 +169,7 @@ NOTE: you can define the same behaviour when you use the `ejb-jar.xml`
       descriptor instead of the annotations
 
 The WildFly provides specific annotation `org.jboss.ejb3.annotation.TransactionTimeout`
-which gives you chance to change transaction timeout for particular bean/method.
+which gives you the chance to change transaction timeout for particular bean/method.
```

Comment:
"which gives you the chance to change the transaction timeout for a particular bean/method"